### PR TITLE
sbt 1.1.5

### PR DESCRIPTION
**Pull request by sullis**

### CODE REVIEW

**Comment by sullis** [comment]
@dwijnand

File: build.sbt [hunk 1]
```diff
@@ -10,12 +10,12 @@ organization := "com.dwijnand"
      scmInfo := Some(ScmInfo(url("https://github.com/dwijnand/sbt-dynver"), "scm:git:git@github.com:dwijnand/sbt-dynver.git"))
 
        sbtPlugin           := true
-      sbtVersion in Global := "1.0.0" // must be Global, otherwise ^^ won't change anything
-crossSbtVersions           := List("0.13.17", "1.0.0")
+      sbtVersion in Global := "1.1.5" // must be Global, otherwise ^^ won't change anything
+crossSbtVersions           := List("0.13.17", "1.1.5")
```
Comment (dwijnand) [inline build.sbt]: The use of 1.0.0 is intentional here (and the line above) as it's the minimum version this plugin needs.

File: build.sbt [hunk 2]
```diff
@@ -11,11 +11,11 @@ organization := "com.dwijnand"
 
        sbtPlugin           := true
       sbtVersion in Global := "1.0.0" // must be Global, otherwise ^^ won't change anything
-crossSbtVersions           := List("0.13.17", "1.0.0")
+crossSbtVersions           := List("0.13.17", "1.1.5")
```
Comment (dwijnand) [inline build.sbt]: The use of 1.0.0 here is intentional.

**Comment by dwijnand** [comment]
Superseded by #69

**Pull request closed without merging**